Cargo.toml

```diff
@@ -16,3 +16,6 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [target.'cfg(target_os = "freebsd")'.dependencies]
 libc = "0.2.107"
+
+[target.'cfg(target_os = "macos")'.dependencies]
+libc = "0.2.107"
```

Comment:
This can be folded into the previous few lines.

```toml
[target.'cfg(any(target_os = "macos", target_os = "freebsd"))'.dependencies]
libc = "0.2.107"
```